website/docs/docs/dbt-versions/release-notes/72-Feb-2024/exports-rn.md

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
consider naming this as just `exports.md` . 

i was using `-rn.md` but recently got good feedback that it's a little redundant in the URL `/docs/dbt-versions/release-notes/.../-rn` so have stopped